.github/workflows/hydro-stand.yaml

```diff
@@ -0,0 +1,110 @@
+      FILE: hydro-stand/stand.yaml
+      NAME: ${{ github.event.inputs.customer }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Update App Name
+        shell: bash
+        run: |
+          sed -i 's/.*"name": "example"/  "name": "${{ env.NAME }}"/g' ${{ env.FILE }}
+
+      - name: Update App Namespace
+        shell: bash
+        run: |
+          sed -i 's/.*"namespace": "example"/    "namespace": "${{ env.NAME }}"/g' ${{ env.FILE }}
+
+      - name: Update App Target Revision
+        shell: bash
+        run: |
+          sed -i 's/.*"targetRevision": "HEAD"/    "targetRevision": "${{ github.event.inputs.tag }}"/g' ${{ env.FILE }}
+      
+      - name: Update App's minio endpoint
+        shell: bash
+        run: |
+          sed -i 's/.*hydro-minio.*/        "value": "hydro-minio-${{ env.NAME }}.dev.hydrosphere.io"/g' ${{ env.FILE }}
+      
+      - name: Update App's registry endpoint
+        shell: bash
+        run: |
+          sed -i 's/.*hydro-registry.*/        "value": "hydro-registry-${{ env.NAME }}.dev.hydrosphere.io"/g' ${{ env.FILE }}
+      
+      - name: Update App's grpc endpoint
+        shell: bash
+        run: |
+          sed -i 's/.*hydro-grpc.*/        "value": "hydro-grpc-${{ env.NAME }}.dev.hydrosphere.io"/g' ${{ env.FILE }}
+          
+      - name: Update App's serving endpoint
+        shell: bash
+        run: |
+          sed -i 's/.*hydro-example.*/        "value": "hydro-${{ env.NAME }}.dev.hydrosphere.io"/g' ${{ env.FILE }}
+      
+      - name: Update App's toleration
+        shell: bash
+        run: |
+          sed -i 's/        "value": "example"/        "value": "${{ env.NAME }}"/g' ${{ env.FILE }}
+        
+      - name: Rename App File 
+        shell: bash
+        run: "cp ${{ env.FILE }} hydro-stand/${{ env.NAME }}-stand.yaml"
+
+      - name: Pushes ${{ env.NAME }} App Manifest
+        uses: dmnemec/copy_file_to_another_repo_action@main
+        env:
+          API_TOKEN_GITHUB: ${{ secrets.HYDRO_SERVING_TOKEN }}
+        with:
+          source_file: 'hydro-stand/${{ env.NAME }}-stand.yaml'
+          destination_repo: 'Hydrospheredata/hydro-serving-infrastructure'
+          destination_branch: 'stage/dev'
+          destination_folder: 'dev/apps/'
+          user_email: 'rufusnorbert14@gmail.com'
+          user_name: 'rufusnufus'
+          commit_message: 'Add ${{ env.NAME }} stand manifest'
+
+  delete_stand:
+    if: github.event.inputs.delete_stand == 'yes'
+    name: Delete stand
+    runs-on: ubuntu-20.04
+    env:
+      NAME: ${{ github.event.inputs.customer }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2 
+        with:
+          repository: Hydrospheredata/hydro-serving-infrastructure
+          ref: stage/dev
+          token: ${{ secrets.HYDRO_SERVING_TOKEN }}
+      
+      - name: Delete ${{ env.NAME }} App Manifest
+        run: |
+          rm dev/apps/${{ env.NAME }}-stand.yaml
```

Comment:
Maybe more effective use archive folder for the history? Some customers can be back for long time